### PR TITLE
Add JetBrains ignores to .gitgnore

### PR DESCRIPTION
**Pull request by GaryDeco**
Added ignores for JetBrains Rider

### CODE REVIEW

**Comment by djungelorm** [comment]
Did you intend to mark this as a draft?

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,11 @@ __pycache__/
 
 # MSBuild Binary and Structured Log
 *.binlog
+
+# JetBrains
+.idea
+.idea/**/workspace.xml
```
Comment (djungelorm) [inline .gitignore:35]: Isn't just having `.idea` enough? The other lines are redundant?

Comment (GaryDeco) [reply .gitignore:35]: It's the standard JetBrains ignores from the GitHub template. But yes I added the top entry to cover all cases. Would you like me to remove the rest? I'll have to do that later when I can get to my PC. I usually just throw one .gitignore in my projects for all major IDEs and languages where I don't have to go back and update it 50 times.

Comment (djungelorm) [reply .gitignore:35]: OK. `.idea` will also match what the other lines match, so can be removed.

I can update the PR now then feel free to merge at your convenience.

**Review by djungelorm — APPROVED** [review]

**Review by GaryDeco — COMMENTED** [review]
Thank you!

**Pull request merged**